test/system/projects/samples_test.rb

```diff
@@ -1548,6 +1548,81 @@ def retrieve_puids
       end
     end
 
+    test 'can update metadata value that is not from an analysis' do
```

Comment:
Could you add a test case for a user who doesn't have permission to edit the metadata